share/spice/isslocation/isslocation.js

```diff
@@ -0,0 +1,38 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_isslocation = function(api_result) {
+        
+       if(!api_result || api_result !== "success") {
```

Comment:
This exits because you're comparing `api_result` with "success", but you should be comparing `api_result.message` instead.